src/tasks.ts

```diff
@@ -40,10 +40,28 @@ export const reportResults = async (result: Result): Promise<void> => {
 
 export const runTask = async (task: Commands): Promise<void> => {
   switch (task) {
-    case Commands.CreateGitIgnoreBackup:
-      return runCommand("sed -i.bak -e '/dist/d' .gitignore");
     case Commands.RemoveNpmrc:
+      /**
+       * Remove .npmrc file from the repository during release where
+       * @ridedott/semantic-release-action action is used.
+       */
+
       return runCommand('rm -rf .npmrc');
+    case Commands.PreInstallPlugins:
+      /**
+       * Install semantic-release and semantic-release
+       * plugins inside of the repository during release where
+       * @ridedott/semantic-release-action action is used.
```

Comment:
Can we reformat the above comment so we have more words in one line? It's well below 80 characters. We should probably find/write a plugin that can do it automatically and include it in `@ridedott/eslint-config`, that's a shame Prettier doesn't do that.

Reply:
Yes, will create an issue

Reply:
https://emtransit.atlassian.net/browse/CP-963?atlOrigin=eyJpIjoiZjMwYjY2OGE0YzU5NDc5NTg5MWIwOTUwYjM4ODZmZWYiLCJwIjoiaiJ9

Reply:
Let's not link resources that are not accessible to everyone. We should open an issue here, so it's visible to anyone. It's an Open Source project.